site/playroom.config.js

```diff
@@ -1,4 +1,6 @@
 const path = require('path');
+// Todo - shouldn't need eslint disable
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Any ideas on what is happening here?

Reply:
Not sure currently, we had the same issue in Metropolis. Even though the IDE is able to find the plugin it shows a lint error

Reply:
I think it's an `eslint-plugin-import` issue https://github.com/import-js/eslint-plugin-import/issues/1810.

Supposedly using `eslint-import-resolver-typescript` fixes it, but we already use it in `eslint-config-seek`, so I'm not sure why it's not working. Could be because this is in a `.js` file, so the typescript resolver doesn't get used.

Not worth fussing over IMO.